docs/user-guide/vpcs.md

```diff
@@ -197,3 +198,63 @@ spec:
   - from: 1000
     to: 2999
 ```
+
+## Mode
+
+VPCs can operate in two modes: L2VNI and L3VNI. L2VNI is the default mode of
+operation and represents the conventional functionality. L3VNI is designed
+for switches that lack the hardware support for L2VNI.
+
+### L2VNI Mode
+
+This is the conventional multi-tenant network virtualization mode. It is the
+default option for VPCs.
+
+
+### L3VNI Mode
+
+In L3VNI mode, the switches are configured to exclusively route unicast traffic.
+This enables multi-tenancy inside of a fabric, even with switches of mixed
+capabilities. The [DS5000](../reference/profiles.md#celestica-ds5000) is an
+L3-only leaf and VPCs attached to this switch must be in L3VNI mode.
```

Comment:
We just found out thant L3VNI mode does not work with ESLAG. I'm adding validation to prevent an L3VNI VPC to be attached on an ESLAG connection, but we should also document it, and this looks like a good place to do so